### ISSUE TITLE
The insert statement is abnormal."ERROR:  Cannot invoke "org.antlr.v4.runtime.tree.ParseTree.accept(org.antlr.v4.runtime.tree.ParseTreeVisitor)" "

### ISSUE THREAD

**Issue by peilinqian** [apache/shardingsphere#21162]
### Which version of ShardingSphere did you use?
we find java version: java17, full_version=17.0.1
ShardingSphere-5.2.0

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy

### Expected behavior
The insert statement is normal.

### Actual behavior
The insert statement is abnormal.

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
```
drop table customer;							 
CREATE TABLE CUSTOMER ( C_CUSTKEY INTEGER NOT NULL,
C_NAME VARCHAR(25) NOT NULL,
C_ADDRESS VARCHAR(40) NOT NULL,
C_NATIONKEY INTEGER NOT NULL,
C_PHONE CHAR(15) NOT NULL,
C_ACCTBAL DECIMAL(15,2)   NOT NULL,
C_MKTSEGMENT CHAR(10) NOT NULL,
C_COMMENT VARCHAR(117) NOT NULL);

INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (107451, 'Customer#000107451', 'jZiY8FZz1pESpfE0ZvicC4f9evS1lrF', 24, '34-474-838-7669', -709.94, 'FURNITURE ', 'ans. furiously final theodolites are carefully. carefully final theodolites alongside of the');   --abnormal

INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c'); --normal
```
**background log**
```
[ERROR] 2022-09-24 10:10:28.595 [epollEventLoopGroup-3-7] o.a.s.p.f.c.CommandExecutorTask - Exception occur:
java.lang.NullPointerException: Cannot invoke "org.antlr.v4.runtime.tree.ParseTree.accept(org.antlr.v4.runtime.tree.ParseTreeVisitor)" because "tree" is null
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createCommonBinaryOperationSegment(OpenGaussStatementSQLVisitor.java:348)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitAExpr(OpenGaussStatementSQLVisitor.java:303)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitAExpr(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$AExprContext.accept(OpenGaussStatementParser.java:14788)
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:770)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:767)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:767)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.createInsertValuesSegments(OpenGaussStatementSQLVisitor.java:758)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsertRest(OpenGaussStatementSQLVisitor.java:696)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsertRest(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$InsertRestContext.accept(OpenGaussStatementParser.java:3231)
        at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:18)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsert(OpenGaussStatementSQLVisitor.java:632)
        at org.apache.shardingsphere.sql.parser.opengauss.visitor.statement.impl.OpenGaussStatementSQLVisitor.visitInsert(OpenGaussStatementSQLVisitor.java:194)
        at org.apache.shardingsphere.sql.parser.autogen.OpenGaussStatementParser$InsertContext.accept(OpenGaussStatementParser.java:3089)
        at org.apache.shardingsphere.sql.parser.api.SQLVisitorEngine.visit(SQLVisitorEngine.java:54)
        at org.apache.shardingsphere.infra.parser.sql.SQLStatementParserExecutor.parse(SQLStatementParserExecutor.java:48)
        at org.apache.shardingsphere.infra.parser.sql.SQLStatementParserEngine.parse(SQLStatementParserEngine.java:47)
        at org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine.parse(ShardingSphereSQLParserEngine.java:58)
        at org.apache.shardingsphere.proxy.backend.handler.ProxyBackendHandlerFactory.newInstance(ProxyBackendHandlerFactory.java:86)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.query.simple.OpenGaussComQueryExecutor.<init>(OpenGaussComQueryExecutor.java:72)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecutorFactory.getCommandExecutor(OpenGaussCommandExecutorFactory.java:114)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecutorFactory.newInstance(OpenGaussCommandExecutorFactory.java:76)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.OpenGaussCommandExecuteEngine.getCommandExecutor(OpenGaussCommandExecuteEngine.java:62)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:108)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:77)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

### Example codes for reproduce this issue (such as a github link).
```
rules:
- !SHARDING
  tables:
    lineitem:
      actualDataNodes: ds_${0..1}.lineitem_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: l_partkey
          shardingAlgorithmName: lineitem-database-inline
      tableStrategy:
        standard:
          shardingColumn: l_orderkey
          shardingAlgorithmName: lineitem-table-inline
    customer:
      actualDataNodes: ds_${0..1}.customer_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: c_custkey
          shardingAlgorithmName: customer-database-inline
      tableStrategy:
        standard:
          shardingColumn: c_nationkey
          shardingAlgorithmName: customer-table-inline
    nation:
      actualDataNodes: ds_${0..1}.nation_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: n_nationkey
          shardingAlgorithmName: nation-database-inline
      tableStrategy:
        standard:
          shardingColumn: n_regionkey
          shardingAlgorithmName: nation-table-inline
    orders:
      actualDataNodes: ds_${0..1}.orders_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: o_orderkey
          shardingAlgorithmName: orders-database-inline
      tableStrategy:
        standard:
          shardingColumn: o_custkey
          shardingAlgorithmName: orders-table-inline
    part:
      actualDataNodes: ds_${0..1}.part_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: p_partkey
          shardingAlgorithmName: part-database-inline
      tableStrategy:
        standard:
          shardingColumn: p_size
          shardingAlgorithmName: part-table-inline
    partsupp:
      actualDataNodes: ds_${0..1}.partsupp_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: ps_partkey
          shardingAlgorithmName: partsupp-database-inline
      tableStrategy:
        standard:
          shardingColumn: ps_suppkey
          shardingAlgorithmName: partsupp-table-inline
    region:
      actualDataNodes: ds_${0..1}.region
      databaseStrategy:
        standard:
          shardingColumn: r_regionkey
          shardingAlgorithmName: region-database-inline
    supplier:
      actualDataNodes: ds_${0..1}.supplier_${0..1}
      databaseStrategy:
        standard:
          shardingColumn: s_suppkey
          shardingAlgorithmName: supplier-database-inline
      tableStrategy:
        standard:
          shardingColumn: s_nationkey
          shardingAlgorithmName: supplier-table-inline
  defaultDatabaseStrategy:
    none:
  defaultTableStrategy:
    none:

  shardingAlgorithms:
    lineitem-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${l_partkey % 2}
    lineitem-table-inline:
      type: INLINE
      props:
        algorithm-expression: lineitem_${l_orderkey % 2}
    customer-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${c_custkey % 2}
    customer-table-inline:
      type: INLINE
      props:
        algorithm-expression: customer_${c_nationkey % 2}
    nation-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${n_nationkey % 2}
    nation-table-inline:
      type: INLINE
      props:
        algorithm-expression: nation_${n_regionkey % 2}
    orders-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${o_orderkey % 2}
    orders-table-inline:
      type: INLINE
      props:
        algorithm-expression: orders_${o_custkey % 2}
    part-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${p_partkey % 2}
    part-table-inline:
      type: INLINE
      props:
        algorithm-expression: part_${p_size % 2}
    partsupp-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${ps_partkey % 2}
    partsupp-table-inline:
      type: INLINE
      props:
        algorithm-expression: partsupp_${ps_suppkey % 2}
    region-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${r_regionkey % 2}
    supplier-database-inline:
      type: INLINE
      props:
        algorithm-expression: ds_${s_suppkey % 2}
    supplier-table-inline:
      type: INLINE
      props:
        algorithm-expression: supplier_${s_nationkey % 2}
schemaName: tpch_db_2
```

**Comment by TeslaCN** [comment]
I was using c3596089f4bfccd5d2b02b164c203f72e5d779be.

I found the second insert didn't work neither.

![image](https://user-images.githubusercontent.com/20503072/192077373-b67ee258-5fb9-4817-b3b7-d8e7b8f17e0a.png)


```
[INFO ] 2022-09-24 11:01:00.756 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Logic SQL: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');
[INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - SQLStatement: OpenGaussInsertStatement(super=InsertStatement(super=AbstractSQLStatement(parameterCount=0, parameterMarkerSegments=[], commentSegments=[]), table=SimpleTableSegment(tableName=TableNameSegment(startIndex=12, stopIndex=19, identifier=IdentifierValue(value=customer, quoteCharacter=NONE)), owner=Optional.empty, alias=Optional.empty), insertColumns=Optional[InsertColumnsSegment(startIndex=21, stopIndex=108, columns=[ColumnSegment(startIndex=22, stopIndex=30, identifier=IdentifierValue(value=c_custkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=33, stopIndex=38, identifier=IdentifierValue(value=c_name, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=41, stopIndex=49, identifier=IdentifierValue(value=c_address, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=52, stopIndex=62, identifier=IdentifierValue(value=c_nationkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=65, stopIndex=71, identifier=IdentifierValue(value=c_phone, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=74, stopIndex=82, identifier=IdentifierValue(value=c_acctbal, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=85, stopIndex=96, identifier=IdentifierValue(value=c_mktsegment, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=99, stopIndex=107, identifier=IdentifierValue(value=c_comment, quoteCharacter=NONE), owner=Optional.empty)])], insertSelect=Optional.empty, values=[InsertValuesSegment(startIndex=117, stopIndex=265, values=[LiteralExpressionSegment(startIndex=118, stopIndex=120, literals=720), LiteralExpressionSegment(startIndex=123, stopIndex=142, literals=Customer#000000720), LiteralExpressionSegment(startIndex=145, stopIndex=176, literals=8shFEL7J9sq1NJXR8dixBqaTO,kbSx), LiteralExpressionSegment(startIndex=179, stopIndex=180, literals=22), LiteralExpressionSegment(startIndex=183, stopIndex=199, literals=32-575-838-4260), LiteralExpressionSegment(startIndex=202, stopIndex=208, literals=5357.32), LiteralExpressionSegment(startIndex=211, stopIndex=222, literals=HOUSEHOLD ), LiteralExpressionSegment(startIndex=225, stopIndex=264, literals=furiously above the furiously ironic c)])]), withSegment=Optional.empty, onDuplicateKeyColumnsSegment=null)
[INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Actual SQL: ds_0 ::: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES org.apache.shardingsphere.sharding.rewrite.token.pojo.ShardingInsertValuesToken@c4408a3;
```

**Comment by peilinqian** [comment]
> I was using [c359608](https://github.com/apache/shardingsphere/commit/c3596089f4bfccd5d2b02b164c203f72e5d779be).
> 
> I found the second insert didn't work neither.
> 
> ![image](https://user-images.githubusercontent.com/20503072/192077373-b67ee258-5fb9-4817-b3b7-d8e7b8f17e0a.png)
> 
> ```
> [INFO ] 2022-09-24 11:01:00.756 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Logic SQL: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');
> [INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - SQLStatement: OpenGaussInsertStatement(super=InsertStatement(super=AbstractSQLStatement(parameterCount=0, parameterMarkerSegments=[], commentSegments=[]), table=SimpleTableSegment(tableName=TableNameSegment(startIndex=12, stopIndex=19, identifier=IdentifierValue(value=customer, quoteCharacter=NONE)), owner=Optional.empty, alias=Optional.empty), insertColumns=Optional[InsertColumnsSegment(startIndex=21, stopIndex=108, columns=[ColumnSegment(startIndex=22, stopIndex=30, identifier=IdentifierValue(value=c_custkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=33, stopIndex=38, identifier=IdentifierValue(value=c_name, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=41, stopIndex=49, identifier=IdentifierValue(value=c_address, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=52, stopIndex=62, identifier=IdentifierValue(value=c_nationkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=65, stopIndex=71, identifier=IdentifierValue(value=c_phone, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=74, stopIndex=82, identifier=IdentifierValue(value=c_acctbal, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=85, stopIndex=96, identifier=IdentifierValue(value=c_mktsegment, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=99, stopIndex=107, identifier=IdentifierValue(value=c_comment, quoteCharacter=NONE), owner=Optional.empty)])], insertSelect=Optional.empty, values=[InsertValuesSegment(startIndex=117, stopIndex=265, values=[LiteralExpressionSegment(startIndex=118, stopIndex=120, literals=720), LiteralExpressionSegment(startIndex=123, stopIndex=142, literals=Customer#000000720), LiteralExpressionSegment(startIndex=145, stopIndex=176, literals=8shFEL7J9sq1NJXR8dixBqaTO,kbSx), LiteralExpressionSegment(startIndex=179, stopIndex=180, literals=22), LiteralExpressionSegment(startIndex=183, stopIndex=199, literals=32-575-838-4260), LiteralExpressionSegment(startIndex=202, stopIndex=208, literals=5357.32), LiteralExpressionSegment(startIndex=211, stopIndex=222, literals=HOUSEHOLD ), LiteralExpressionSegment(startIndex=225, stopIndex=264, literals=furiously above the furiously ironic c)])]), withSegment=Optional.empty, onDuplicateKeyColumnsSegment=null)
> [INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Actual SQL: ds_0 ::: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES org.apache.shardingsphere.sharding.rewrite.token.pojo.ShardingInsertValuesToken@c4408a3;
> ```

![image](https://user-images.githubusercontent.com/97432411/192077430-dfa7c7f4-fe4c-4eb0-92f6-88509f9d14e5.png)

**Comment by peilinqian** [comment]
> > I was using [c359608](https://github.com/apache/shardingsphere/commit/c3596089f4bfccd5d2b02b164c203f72e5d779be).
> > I found the second insert didn't work neither.
> > ![image](https://user-images.githubusercontent.com/20503072/192077373-b67ee258-5fb9-4817-b3b7-d8e7b8f17e0a.png)
> > ```
> > [INFO ] 2022-09-24 11:01:00.756 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Logic SQL: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');
> > [INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - SQLStatement: OpenGaussInsertStatement(super=InsertStatement(super=AbstractSQLStatement(parameterCount=0, parameterMarkerSegments=[], commentSegments=[]), table=SimpleTableSegment(tableName=TableNameSegment(startIndex=12, stopIndex=19, identifier=IdentifierValue(value=customer, quoteCharacter=NONE)), owner=Optional.empty, alias=Optional.empty), insertColumns=Optional[InsertColumnsSegment(startIndex=21, stopIndex=108, columns=[ColumnSegment(startIndex=22, stopIndex=30, identifier=IdentifierValue(value=c_custkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=33, stopIndex=38, identifier=IdentifierValue(value=c_name, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=41, stopIndex=49, identifier=IdentifierValue(value=c_address, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=52, stopIndex=62, identifier=IdentifierValue(value=c_nationkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=65, stopIndex=71, identifier=IdentifierValue(value=c_phone, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=74, stopIndex=82, identifier=IdentifierValue(value=c_acctbal, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=85, stopIndex=96, identifier=IdentifierValue(value=c_mktsegment, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=99, stopIndex=107, identifier=IdentifierValue(value=c_comment, quoteCharacter=NONE), owner=Optional.empty)])], insertSelect=Optional.empty, values=[InsertValuesSegment(startIndex=117, stopIndex=265, values=[LiteralExpressionSegment(startIndex=118, stopIndex=120, literals=720), LiteralExpressionSegment(startIndex=123, stopIndex=142, literals=Customer#000000720), LiteralExpressionSegment(startIndex=145, stopIndex=176, literals=8shFEL7J9sq1NJXR8dixBqaTO,kbSx), LiteralExpressionSegment(startIndex=179, stopIndex=180, literals=22), LiteralExpressionSegment(startIndex=183, stopIndex=199, literals=32-575-838-4260), LiteralExpressionSegment(startIndex=202, stopIndex=208, literals=5357.32), LiteralExpressionSegment(startIndex=211, stopIndex=222, literals=HOUSEHOLD ), LiteralExpressionSegment(startIndex=225, stopIndex=264, literals=furiously above the furiously ironic c)])]), withSegment=Optional.empty, onDuplicateKeyColumnsSegment=null)
> > [INFO ] 2022-09-24 11:01:00.757 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Actual SQL: ds_0 ::: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES org.apache.shardingsphere.sharding.rewrite.token.pojo.ShardingInsertValuesToken@c4408a3;
> > ```
> 
> ![image](https://user-images.githubusercontent.com/97432411/192077430-dfa7c7f4-fe4c-4eb0-92f6-88509f9d14e5.png)


**Your problem is another issue**
https://github.com/apache/shardingsphere/issues/18456

**Comment by TeslaCN** [comment]
This could be reproduced by 

```sql
insert into t (id) values (-1);
```

![image](https://user-images.githubusercontent.com/20503072/192081394-04f570d3-4947-4c90-a9f6-ef4128a09d30.png)

**Comment by TeslaCN** [comment]
Hi @strongduanmu 
I found this issue may be caused by #20238. Could you take a look at this?

The following code relates to this issue.

https://github.com/apache/shardingsphere/blob/32aa357e140c33292cb900f8a33f09aeae5660dd/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/src/main/java/org/apache/shardingsphere/sql/parser/opengauss/visitor/statement/impl/OpenGaussStatementSQLVisitor.java#L323-L325

**Comment by strongduanmu** [comment]
@TeslaCN Thank you for your remind, I will investigate this issue.

**Comment by strongduanmu** [comment]
I have test with d8f7ab5872bbd7251cda72ed5d1d5cb4e2394033, these sqls all run successfully.

<img width="1914" alt="image" src="https://user-images.githubusercontent.com/10829171/192179224-2bcb54ab-ad26-4cab-803d-a6edb3d1b150.png">

**Comment by TeslaCN** [comment]
> I have test with [d8f7ab5](https://github.com/apache/shardingsphere/commit/d8f7ab5872bbd7251cda72ed5d1d5cb4e2394033), these sqls all run successfully.
> 
> <img alt="image" width="1914" src="https://user-images.githubusercontent.com/10829171/192179224-2bcb54ab-ad26-4cab-803d-a6edb3d1b150.png">

The key is negative number. You may try the following SQL:
```sql
insert into t (id) values (-1);
```

**Comment by strongduanmu** [comment]
Thank you for your remind, I will try again.

**Comment by wsm12138** [comment]
fixed by 4114e7ee4cbe5923c2b403a3e86d1f23355cadf3
 
<img width="1826" alt="image" src="https://user-images.githubusercontent.com/86462784/192256023-3c4713b1-328e-421f-9328-a56a0ee28822.png">

**Comment by peilinqian** [comment]
```
tpch_db=> CREATE TABLE CUSTOMER ( C_CUSTKEY INTEGER NOT NULL,
tpch_db(> C_NAME VARCHAR(25) NOT NULL,
tpch_db(> C_ADDRESS VARCHAR(40) NOT NULL,
tpch_db(> C_NATIONKEY INTEGER NOT NULL,
tpch_db(> C_PHONE CHAR(15) NOT NULL,
tpch_db(> C_ACCTBAL DECIMAL(15,2)   NOT NULL,
tpch_db(> C_MKTSEGMENT CHAR(10) NOT NULL,
tpch_db(> C_COMMENT VARCHAR(117) NOT NULL);
CREATE TABLE
tpch_db=>
tpch_db=> INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (107451, 'Customer#000107451', 'jZiY8FZz1pESpfE0ZvicC4f9evS1lrF', 24, '34-474-838-7669', -709.94, 'FURNITURE ', 'ans. furiously final theodolites are carefully. carefully final theodolites alongside of the');   --abnormal
INSERT 0 1
tpch_db=>
tpch_db=> INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c'); --normal
INSERT 0 1

```
